sbom-utility-scripts/scripts/base-images-sbom-script/app/test_base_images_sbom_script.py

```diff
@@ -516,6 +517,131 @@ def test_main_input_sbom_does_not_contain_formulation(tmp_path, mocker):
     assert expected_output["formulation"] == sbom["formulation"]
 
 
+@pytest.fixture
+def isodate():
+    with patch("datetime.datetime") as mock_datetime:
+        mock_datetime.now.return_value.isoformat.return_value = "2021-07-01T00:00:00Z"
```

Comment:
This doesn't match the actual output format of `isoformat()`

Reply:
Yeah, it is not. But follows the specification. Should we rename it? I chose isodate as it's format which resemble is the most.

Reply:
IMO we should the format that matches the specification (mainly in the production code, not just in test mocks)